runtime/compiler/x/codegen/J9TreeEvaluator.cpp

```diff
@@ -573,12 +574,331 @@ static TR::Instruction *generatePrefetchAfterHeaderAccess(TR::Node
    return instr;
    }
 
+// General float/double convert to long
```

Comment:
Can you add `(for 32-bit)` or words to that effect in this comment?  It will make it clear that this conversion only applies to 32-bit without having to grok the code.